[DOC] Update computing section (#73)
poldracklab/poldracklab.github.io: labguide/computing/sherlock/access-and-resources.md

+---------------------------------------------------------------------------+
          HOME |   9.4GB /  15.0GB [||||||     62%] |      - /      - (  -%)
    GROUP_HOME | 562.6GB /   1.0TB [|||||      56%] |      - /      - (  -%)
       SCRATCH |  65.0GB / 100.0TB [            0%] | 143.8K /  20.0M (  0%)
 GROUP_SCRATCH | 172.2GB / 100.0TB [            0%] |  53.4K /  20.0M (  0%)
           OAK |  30.8TB / 240.0TB [|          12%] |   6.6M /  36.0M ( 18%)
+---------------------------------------------------------------------------+
```

`sh_quota` is a Sherlock-specific command that provides a general overview for all partitions a user has access to.
[Documentation is provided on their wiki](https://www.sherlock.stanford.edu/docs/storage/?h=sh_quota#checking-quotas).
When your home directory begins to get filled, it may be valuable to consider moving files to scratch directories, or group directories.
`HOME`, `GROUP_HOME`, and `OAK` are persistent storage; `*SCRATCH` directories are subject to purging.

Another useful tool is the disk usage command `du`.
A useful and more interacrtive version of this command is `ncdu`.

> To use `ncdu`, add the following line to the bottom of your `~/.bash_profile`, which will load the `ncdu` module each time you log in to Sherlock:
> 
> ```bash
> $ ml system ncdu
> ```

Reviewer: ```suggestion
To use `ncdu`, add `ml system ncdu` to the bottom of your `~/.bash_profile`, which will load the `ncdu` module each time you log in to Sherlock.
You can add and access it with the following commands:

````bash
$ echo "ml system ncdu" >> ~/.bash_profile
$ source ~/.bash_profile
```
````

> In future login session, you can access the `ncdu` command via

Reviewer: ```suggestion
In future login sessions, `ncdu` will be accessible without calling `source`.
Use the `ncdu` command via
```
